hs_core/views/__init__.py

```diff
@@ -2510,7 +2511,7 @@ def get_context_data(self, **kwargs):
 
         if self.request.user.is_authenticated:
             group_members = g.gaccess.members
-            u = User.objects.get(pk=self.request.user.id)
+            u = User.objects.select_related("uaccess").get(pk=self.request.user.id)
```

Comment:
if we `select_related` here, does that impact line 2555 `"profile_user": u,`

Reply:
Same answer as above.

Reply:
thanks